### ISSUE TITLE
Usage of DelegatedFactory seems not correct in an example

### ISSUE THREAD

**Issue by ojob** [ets-labs/python-dependency-injector#285]
I'm reading the docs, and it seems to me that one of the examples is not correct:

https://github.com/ets-labs/python-dependency-injector/blob/2b30e172d13337db8b88ffb9c167eedf283a2840/examples/providers/factory_delegation.py#L29

Indeed, it seems to me that this line should simple read, as for the other examples in the same file:

```python
users_factory = providers.Factory(
#                         ^ no need to use a Delegated Factory here
```

Is my understanding correct?

**Comment by rmk135** [comment]
Hi @ojob , yep, you're right. It should be just `Factory` provider.

**Comment by rmk135** [comment]
Fixed in [3.32.2](http://python-dependency-injector.ets-labs.org/main/changelog.html#id2).
Thank you for finding and reporting the issue.

**Comment by ojob** [comment]
You're welcome! And contrags for the reactivity in fixing it.
